node/send_receive/lib/eventdata.js

```diff
@@ -6,9 +6,10 @@
 /**
  * @class EventData
  * @classdesc Constructs a {@linkcode EventData} object.
- * @param {String}  body   The event payload as a byte array.
+ * @param {object} link The receiver link for providing settlment methods.
+ * @param {object}  message The message object containing body and messageAnnotations.
  */
-function EventData(body, systemProperties) {
+function EventData(link, message) {
```

Comment:
Ok, cool, that's the direction I was going down before I realized you'd said "client" :) I'll continue.